aries_cloudagent/config/argparse.py

```diff
@@ -872,31 +872,51 @@ def get_settings(self, args: Namespace) -> dict:
         if args.no_ledger:
             settings["ledger.disabled"] = True
         else:
-            configured = False
+            single_configured = False
+            multi_configured = False
```

Comment:
add `pool_name_updated = False`

Reply:
This and the following changes can be a possible solution to `ledger.pool_name` issue.